components/library/src/taxonomies/mod.rs

```diff
@@ -121,8 +138,23 @@ impl Taxonomy {
         for (name, pages) in items {
             sorted_items.push(TaxonomyItem::new(&name, &kind, config, pages, library));
         }
-        sorted_items.sort_by(|a, b| a.name.cmp(&b.name));
+        // make sure we have stabel sorting when slugs are equal
```

Comment:
stable

Reply:
I assume you mean that you prefer the original bit since that is already stable, which is fine, I'll revert it when I make the change

Reply:
Edit, sorry I just realised sorting by slug first is actually necessary since the deduping otherwise fails. Tests fail with the original sorting